packages/teleport-project-generator/src/file-handlers.ts

```diff
@@ -43,6 +47,40 @@ export const createComponent = async (
   return strategy.components.generator.generateComponent(componentUIDL, options)
 }
 
+export const createComponentModule = async (uidl: ProjectUIDL, strategy: ProjectStrategy) => {
```

Comment:
maybe createModule?

Reply:
We need to generate three different kinds of modules for Angular, one for root and second for pages and for components, just to differentiate the three, I named them like this. How about adding a comment on the top?

Reply:
ah, got it, makes sense